### PR TITLE
Add DigitalOcean as an adopter

### PR DESCRIPTION
**Pull request by mamercad**
DigitalOcean ❤️ StackStorm

### CODE REVIEW

**Review by cognifloyd — APPROVED** [review]

File: ADOPTERS.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ This is an alphabetical list of known [StackStorm](https://stackstorm.com/) adop
...
 * [Bitovi](https://www.bitovi.com/) - Consulting company, implemented an Automation solution based on StackStorm API with HA capabilities and custom UI for a Fortune top 10 organization. [[ Blog ](https://www.bitovi.com/blog/stackstorm-solves-devops-automation-for-enterprise-client)] [[ Case study ](https://stackstorm.com/case-study-bitovi/)]
 * [DMM.com](https://dmm-corp.com/en/) - Large content provider in Japan. StackStorm is used in Operations helping to maintain online services and development at scale. [[ Case study ](https://stackstorm.com/case-study-dmm/)]
+* [DigitalOcean](https://www.digitalocean.com/about) - DigitalOcean simplifies cloud computing so builders can spend more time creating software that changes the world.
```
Comment (armab) [inline ADOPTERS.md:8]: That describes well what DO does.
@mamercad Maybe you could add a few more words on how StackStorm is used at DO?

Comment (mamercad) [reply ADOPTERS.md:8]: > That describes well what DO does 👍 @mamercad Maybe you could add a few more words on how StackStorm is used at DO? It's definitely something that the st2 community would be interested to get an idea of.

Ah yes, will do :loading:

File: ADOPTERS.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ This is an alphabetical list of known [StackStorm](https://stackstorm.com/) adop
...
 * [Bitovi](https://www.bitovi.com/) - Consulting company, implemented an Automation solution based on StackStorm API with HA capabilities and custom UI for a Fortune top 10 organization. [[ Blog ](https://www.bitovi.com/blog/stackstorm-solves-devops-automation-for-enterprise-client)] [[ Case study ](https://stackstorm.com/case-study-bitovi/)]
 * [DMM.com](https://dmm-corp.com/en/) - Large content provider in Japan. StackStorm is used in Operations helping to maintain online services and development at scale. [[ Case study ](https://stackstorm.com/case-study-dmm/)]
+* [DigitalOcean](https://www.digitalocean.com/about) - DigitalOcean simplifies cloud computing so builders can spend more time creating software that changes the world. Internally, StackStorm is used as a consistent frontend to our numerous operational tools, and it also plays the part of the orchestration and automation engine driving the machine lifecycle of our vast fleet of machines spread across the globe.
```
Comment (armab) [inline ADOPTERS.md:8]: Nice stuff!
Thanks for sharing :+1: 

We'd love to see a blog someday at DigitalOcean about that use case!

**Review by armab — APPROVED** [review]
Happy to see StackStorm's adoption at DigitalOcean and thanks for your involvement in the Open Source community!

File: ADOPTERS.md (hunk 2 above)
Comment (mamercad) [reply ADOPTERS.md:8]: > Nice stuff! Thanks for sharing 👍
> 
> We'd love to see a blog someday at DigitalOcean about that use case!

Ah yes, I will do my best to put one together when I can -- StackStorm's been a huge force multiplier for us!

**Pull request merged**